resources/META-INF/plugin.xml

```diff
@@ -49,9 +49,7 @@
     <!-- Add your actions here -->
     <group id="me.lotabout.codegenerator.action.CodeGeneratorMenu" class="me.lotabout.codegenerator.action.CodeGeneratorGroup" text="CodeGenerator"
            description="Code Generator" popup="true">
-      <add-to-group group-id="EditorPopupMenu2" anchor="after" relative-to-action="GenerateGroup"/>
-      <add-to-group group-id="ProjectViewPopupMenu" anchor="after" relative-to-action="ProjectViewPopupMenuRefactoringGroup"/>
```

Comment:
@aesy Please keep this one. What it does actually is to add the code generator menu to the right click menu of the project view. It is for "class" level templates.